docs/how-to/extract-a-backup.md

```diff
@@ -45,6 +45,22 @@ in the right place before running the command—or see below about the
 `--destination` flag.
 
 
+## Extract the configuration file used to create an archive
```

Comment:
I feel like this is at a odd spot in flow of the page, even though I think I see why you put it first-ish: Because `bootstrap` needs to come first on a fresh system. Anyway, the previous section is about `extract`ing individual files from an existing repo, and the section after this new one is about a variant of that.. specifying a single repository to use with `extract`. Neither of which have to do with bootstrapping!

So here's an alternate idea on where to put your new section such that the whole page flows a little better:

 * Start the page with a brief intro paragraph: "When the worst happens ..." etc etc.
 * Then insert your new section, perhaps prefaced with: "If you're extracting to a fresh system that doesn't have borgmatic configuration files ..." or whatever.
 * And then the next section can dive into the existing `extract` docs with another preface: "If you already have borgmatic configuration files (or you've just bootstrapped them as per above), you can extract your files from a repository. A good way to do that is to use the `rlist` action ..."

If you don't like that approach, then maybe just leave the document mostly alone and then move your new section to the end? That way it doesn't interrupt the flow of talking about the `extract` action across sections?

Reply:
Moved this section to the end since I didn't wanna move a lot of stuff around here.